arviz/data/datasets.py

```diff
@@ -7,6 +7,8 @@
 from urllib.request import urlretrieve
 
 from .io_netcdf import from_netcdf
+from ..rcparams import RcParams
```

Comment:
RcParams i camel case is the name of the class defined, the specific instance to customize arviz is `rcParams` (using the same names as matplotlib which I figured would be less confusing)